style.css

```diff
@@ -0,0 +1,44 @@
+body {
+font-family: 'Shadows Into Light', cursive;
+background-color: hsl(208,30%,90%);
+}
+
+h1 {
+  font-size: 52px;
+  text-transform: capitalize;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 5px;
+  word-spacing: 20px;
+  color: hsl(234,35%,24%); 
+}
+
+h2{
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+  letter-spacing: 2px;
+  word-spacing: 1px;
+  color: hsl(234,35%,22%); 
+}
+
+p{
+  font-family: 'Architects Daughter', cursive;
+  font-size: 20px;
+  font-weight: normal;
```

Comment:
This reiterates the  [default browser styles](https://browserdefaultstyles.com/) for the `p` element and, therefore, can be removed:
```suggestion
```